lib/plugins/logger/lib/LoggerPlugin.js

```diff
@@ -0,0 +1,44 @@
+const Logger = require('./Logger.js')
+
+class LoggerPlugin {
+  didLoadFramework(app) {
+    const {
+      config,
+    } = app
+
+    app.logger = new Logger(config.logger)
+
+    Object.assign(
+      app.context,
```

Comment:
sounds bad to let the plugin freely override the app object lol

Reply:
seems not too bad if it override log and Logger only into context?

> sounds bad to let the plugin freely override the app object lol

seems not too bad if it override log and Logger only into context?